pytket/qir/conversion/api.py

```diff
@@ -244,10 +242,15 @@ def check_circuit(
 
     for creg in circuit.c_registers:
         if creg.size > int_type:
+            hint: str | None = None
+            if creg.name[0:10] == _TEMP_BIT_REG_BASE[0:10]:
```

Comment:
```suggestion
            if creg.name.startswith(_TEMP_BIT_REG_BASE):
```
May need to modify `test_pytket_api_creg_size_3` but I think this is better.